src/pineko/cli/opcard.py

```diff
@@ -28,9 +28,7 @@ def subcommand(pineappl_path, default_card_path, thcard_path, opcard_path):
 
     """
     tcard = yaml.safe_load(pathlib.Path(thcard_path).read_text(encoding="utf-8"))
+    opcard_path = pathlib.Path(opcard_path)
     _x_grid, q2_grid = evolve.write_operator_card_from_file(
         pineappl_path, default_card_path, opcard_path, tcard
     )
-    rich.print(
```

Comment:
This can be done, most of the other print statement are inside the library.

Reply:
yeah I realized that when going further down the review, but the idea still holds - so I'd say let's start with something

Reply:
I think this was done because of the iterative commands, like `op_cards` would not pass from here. 
So the fix require some more changes

Reply:
So the fix is not minimal, as we need to change also `TheoryBuilder.iterate` ecc. 
I think the `opcard` command could be removed as it's a kind of duplication of 
`pineko theory opcard` and then we can fix all the print statement in a separated PR.

Reply:
issue: https://github.com/NNPDF/pineko/issues/186

as for the duplication: I don't think so - because `opcard` is the "file mode" of pineko, while `theory *` is the "theory mode". I'd like to keep the support for the "file mode" - as already said several times, if the default should be "theory mode" or "file mode" is debatable.

Reply:
True, but here the "file mode" requires a theory card and a grid and do not offer any other debug or additional option, so it's really as the "theory mode". For instance this is not the case for `convolve` and `fks` so I see your point.

Reply:
> requires a theory card

indeed and not the whole folder structure (i.e. it operates on files), so it is valuable I'd say